itest/lnd_multi-hop_test.go

```diff
@@ -1936,17 +1950,6 @@ func runMultiHopHtlcAggregation(ht *lntest.HarnessTest,
 
 	expectedTxes := 0
 	switch c {
-	// With the closing transaction confirmed, we should expect Bob's HTLC
-	// timeout transactions to be broadcast due to the expiry being reached.
-	// We will also expect the success transactions, since he learnt the
-	// preimages from Alice. We also expect Carol to sweep her commitment
-	// output.
-	case lnrpc.CommitmentType_LEGACY:
```

Comment:
Removed, because the `commitWithZeroConf` map does only test channel types which have anchor outputs, so this case would have never been hit.